### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by jason-fox**
Add Node 14 to unit-test matrix

### CODE REVIEW

File: test/examples/deviceProvisioning/expectedProvisioningPluginRequest.json [hunk 1]
```diff
@@ -8,35 +8,35 @@
               {
                   "name": "time",
                   "type": "String",
-                  "value": " "
+                  "value": null
```
Comment (fgalan) [inline test/examples/deviceProvisioning/expectedProvisioningPluginRequest.json:11]: Currently unit tests in master are failing:

imagen

Are the changes `" "` to `null` included in this PR the fixes for that?

Just to confirm...

Comment (jason-fox) [reply test/examples/deviceProvisioning/expectedProvisioningPluginRequest.json:11]: Yes

Comment (fgalan) [reply test/examples/deviceProvisioning/expectedProvisioningPluginRequest.json:11]: NTC

File: docs/deprecated.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+-   Docker images corresponding to sigfox-iotagent can be found at
+    [Dockerhub](https://hub.docker.com/r/fiware/sigfox-iotagent/tags/).
+
+The following table provides information about the last sigfox-iotagent version supporting currently removed features:
+
+| **Removed feature**   | **Last sigfox-iotagent version supporting feature** | **That version release date** |
+| --------------------- | ----------------------------------------------- | ----------------------------- |
+| NGSIv1 API            | Not yet defined                                 | Not yet defined               |
+| Support to Node.js v6 | Not yet defined but it will be done by May 2019 | Not yet defined               |
+| Support to Node.js v4 | 1.8.0                                           | December 19th, 2018           |
```
Comment (fgalan) [inline docs/deprecated.md:40]: Looking to https://github.com/telefonicaid/sigfox-iotagent/releases I think the following is more accurate:

```suggestion
| Support to Node.js v4 | 1.0.0                                           | June 13th, 2018                   |
| Support to Node.js v6 | 1.0.0                                           | June 13th, 2018                   |
| Support to Node.js v8 | 1.4.0                                           | April 8th, 2020                     |
```

File: docs/deprecated.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    closed in github.com as soon as they are spotted).
+-   Documentation on deprecated features is removed from the repository documentation. Documentation is still available
+    in the documentation set associated to older versions (in the repository release branches).
+-   Deprecated functionality is eventually removed from sigfox-iotagent. Thus you are strongly encouraged to change your
+    implementations using sigfox-iotagent in order not rely on deprecated functionality.
+
+A list of deprecated features and the version in which they were deprecated follows:
+
+-   Support to NGSI v1.
+-   Support to Node.js v10 in sigfox-iotagent 1.5.0. The use of Node.js v12 is highly recommended.
```
Comment (fgalan) [inline docs/deprecated.md:16]: Looking to https://github.com/telefonicaid/sigfox-iotagent/releases I think the following is more accurate:

```suggestion
-   Support to Node.js v4 and v6 in sigfox-iotagent 1.0.0 (finally removed in 1.1.0)
-   Support to Node.js v8 in sigfox-iotagent 1.4.0 (finally removed in 1.5.0)
-   Support to Node.js v10 in sigfox-iotagent 1.5.0. The use of Node.js v12 is highly recommended.
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**